schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/mail/MailDebug.java

```diff
@@ -31,28 +31,40 @@ private MailDebug() {
     }
 
     static void writeDebug(Message message) {
-        if (!EnvironmentUtil.isDevelopment()) return;
-        if (!Boolean.parseBoolean(System.getProperty(JASIFY_EMAIL_DEBUG_PROP))) {
-            return;
-        }
-        try {
-            message = fixMessageWithAttachment(message);
-            SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
-            String filename = sdf.format(new Date()) + ".eml";
-            ByteArrayOutputStream os = new ByteArrayOutputStream();
-            message.writeTo(os);
-            Attachment attachment = AttachmentHelper.create(filename, "message/rfc822", os.toByteArray());
-            new AttachmentDao().save(attachment);
-            log.info("Recorded e-mail\n\tSubject: {}\n\tURL: {}", message.getSubject(), AttachmentHelper.makeDownloadUrl(attachment));
-        } catch (Exception e) {
-            log.warn("Failed to write debug email", e);
+        if (EnvironmentUtil.isDevelopment()) {
+            if (log.isInfoEnabled() && Boolean.parseBoolean(System.getProperty(JASIFY_EMAIL_DEBUG_PROP))) {
```

Comment:
The `log.isInfoEnabled()` check here is incorrect.  We still want to save the attachment even if we don't log it.